framework/auth/oauth_scopes.py

```diff
@@ -124,8 +128,8 @@ class ComposedScopes(object):
     NODE_ALL_WRITE = NODE_ALL_READ + NODE_METADATA_WRITE + NODE_DATA_WRITE + NODE_ACCESS_WRITE
 
     # Full permissions: all routes intended to be exposed to third party API users
-    FULL_READ = NODE_ALL_READ + USERS_READ + ORGANIZER_READ + GUIDS_READ + (CoreScopes.INSTITUTION_READ, )
-    FULL_WRITE = NODE_ALL_WRITE + USERS_WRITE + ORGANIZER_WRITE + GUIDS_READ
+    FULL_READ = NODE_ALL_READ + USERS_READ + ORGANIZER_READ + GUIDS_READ + (CoreScopes.INSTITUTION_READ, CoreScopes.ADDONS_READ, )
+    FULL_WRITE = NODE_ALL_WRITE + USERS_WRITE + ORGANIZER_WRITE + GUIDS_READ + (CoreScopes.NODE_ADDON_WRITE, )
```

Comment:
> We've been requested to consider, among other use cases, allowing someone to make an application that will allow a user to pick a github repo they control and, with minimal clicking, create an OSF user, link the github account to their OSF account, and create a project with that github repo attached. Possibly some other flourishes in addition.

TLDR: Because we have already started handing these tokens out as the default option, we need to seriously revisit how we're handling `full_read`. Realistically, there should be a totally separate scope for things that touch on sensitive account management.

1. What you describe is an extremely powerful capability- especially given the way github encourages me to mix my personal and professional identity under the same account. I don't want to connect for science, only to find that a client's trade secrets are now vulnerable.

2. At the absolute minimum, we should not give this kind of thing out by default in the absence of a concrete and specific need, and even then, only **after** a careful process is in place. 

3. It is easier to withhold permissions until actually needed than to take them away following a crisis.

4. Making `full_read` less than the full set of powers may admittedly be confusing, but it's a painful consequence of the current system being live and public-facing. In an extreme case, there are mechanisms in place to support deprecating token names entirely.